vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -8370,7 +8370,7 @@ unit:CHF-PER-KiloGM
   qudt:conversionMultiplierSN 1.0E-3 ;
```

Comment:
Since the kilogram is the base SI unit, the conversion multipliers should be 1.0